performance_test/cmake/Modules/FindODB.cmake

```diff
@@ -0,0 +1,141 @@
+# OdbCmake - CMake find module for ODB
+#
+# Copyright (c) 2014 Timo Rothenpieler <btbn@btbn.de>,
```

Comment:
@monidzik It looks like this copyright and a couple others are causing CI to fail the copyright check. There are a few possible fixes:

1. Exclude the files from the copyright check: in CMakeList.txt

```cmake
find_package(ament_lint_auto)
ament_lint_auto_find_test_dependencies()

list(APPEND AMENT_LINT_AUTO_EXCLUDE ament_cmake_copyright)
ament_copyright(<path to all other files>)
```

2. Create a copyright package extension (see the `apex_copyright` package in our internal repo)
3. We install these files in the test environment rather than keep them version controlled here

@esteve Maybe you have other suggestions since you are more familiar with this project and CI?

Reply:
@jpsamper2009 ok, I did 1st possible fix from your list and it helped, thank you!